src/Illuminate/Support/ViewErrorBag.php

```diff
@@ -33,7 +33,7 @@ public function hasBag($key = 'default')
      */
     public function getBag($key)
     {
-        return Arr::get($this->bags, $key, new MessageBag);
+        return Arr::get($this->bags, $key) ?: new MessageBag;
```

Comment:
> would it make sense to change all calls to Arr::get for this reason ?

It doesn't make sense for scalars, simple arrays and already existing objects. But any usage with `new Class()` or especially with `Container::make()` IMO should be replaced.